src/emcpy/io/read_bufr.py

```diff
@@ -0,0 +1,88 @@
+import ncepbufr
```

Comment:
Do we intend to make this a required dependency of EMCPy? Or optional? If the latter, how does one 'import' this class?

Reply:
Great question. I would like to say optional, but we need to make sure to point to the right `PYTHONPATH` for it `.../nceplibs-bufr/install/lib64/python3.7/site-packages`

Reply:
and do we put it in the GitHub CI for unit tests?

Reply:
tagging @aerorahul for input, do you think it is reasonable to make the python BUFR library a dependency of EMCPy? Or should it be separate?

Reply:
No.

Reply:
You could do something like:

```
try:
        import ncepbufr
    except ImportError as error:
        # you can either raise an exception; for e.g.
        raise ImportError('Attempting to load ncepbufr failed')
        # or you can disable use of ncepbufr by setting a logical and using it to wrap around ncepbufr calls.  
        # There may be other ways than using a logical.
```